exploration/text-vs-code.md

```diff
@@ -101,32 +144,62 @@ Which whitespace characters are displayed at runtime should not be surprising.
 
 ## Requirements
 
-Common things should be easy, uncommon things should be possible.
+It should be easy to do simple things; possible to do complex things; and impossible, or at least difficult, to do wrong things.
+
+<details>
+<blockquote>
+APIs should be easy to use and hard to misuse. It should be easy to do simple things; possible to do complex things; and impossible, or at least difficult, to do wrong things.
+
+—<a href="https://www.infoq.com/articles/API-Design-Joshua-Bloch/">Joshua Bloch, 2008</a>, author of <i>Effective Java</i>, etc.
+</blockquote>
+<blockquote>
+The Pit of Success: in stark contrast to a summit, a
+peak, or a journey across a desert to find victory through many trials and
+surprises, we want our customers to simply fall into winning practices by using
+our platform and frameworks. To
+the extent that we make it easy to get into trouble we fail.
+          
+—Rico Mariani, MS Research MindSwap Oct 2003. (<a href="https://learn.microsoft.com/en-us/archive/blogs/brada/the-pit-of-success">restated by Brad Adams</a>, MS CLR and .Net team cofounder)
+</blockquote>
+</details>
 
 Developers and translators should be able to read and write the syntax easily in a text editor.
 
 Translators (and their tools) are not software engineers, so we want our syntax
-to be as simple, robust, and non-fussy as possible.
-Multiple levels of complex nesting should be avoided,
-along with any constructs that require an excessive
-level of precision on the part of non-technical authors.
+to be as simple and robust as possible.
+
+Nesting level is not a requirement.
+People are not parsers, and don't care about the nesting level as a primary concern when reading a message.
+What matters to them is their ability to recognize where a message's pattern starts and where it ends.
```

Comment:
I see what you're trying to do, but the original requirement didn't say "avoid all nesting". It used "should" and was concerned with depth of nesting and message complexity. A well-known shortcoming of MF1 are the plural/select constructs with their complex use of nested `{...}` sequences. The example below is argument and this is (and claims to be!) "not a requirement" in the requirements section. If you want to talk about nesting like this, talk about it in design considerations?

If you want a requirement, then perhaps:

```suggestion
Nesting of text or code should be as simple as possible, but no simpler.
```